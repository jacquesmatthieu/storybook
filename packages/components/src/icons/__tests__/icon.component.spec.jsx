import React from 'react';
import { shallow } from 'enzyme';

import Icon from '../icon.component';

describe('Icon', () => {
  let props;

  beforeEach(() => {
    props = {
      name: 'arrow',
      stroke: 'primary.main',
      theme: {
        palette: {
          primary: {
            main: 'rebeccapurple',
          },
        },
      },
    };
  });

  const getWrapper = () => shallow(<Icon {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
