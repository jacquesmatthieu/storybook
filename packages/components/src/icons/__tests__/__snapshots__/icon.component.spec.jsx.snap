// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render properly 1`] = `
<Arrow
  className=""
  direction="down"
  fill="none"
  size={10}
  stroke="rebeccapurple"
  strokeWidth={2}
/>
`;
